gap/Julia.gi

```diff
@@ -30,3 +30,33 @@ InstallMethod( MapOfFinSets,
     return MapOfFinSets( source, graph, range );
     
 end );
+
+##
+InstallOtherMethod( MorphismConstructor,
+        [ IsCategoryOfSkeletalFinSets, IsSkeletalFiniteSet, IsJuliaObject, IsSkeletalFiniteSet ], 1001,
+        
+  function ( sFinSets, source, graph, range )
+    
+    return MorphismConstructor( sFinSets, source, ConvertJuliaToGAP( graph ), range );
+    
+end );
+
+##
+InstallOtherMethod( MorphismConstructor,
+        [ IsSkeletalFiniteSet, IsJuliaObject, IsSkeletalFiniteSet ], 1001,
+        
+  function ( source, graph, range )
+    
+    return MorphismConstructor( CapCategory( source ), source, graph, range );
+    
+end );
```

Comment:
I would expect those to be covered by the generic handling in CAP. Have you checked if they are?

Reply:
Yes I did and unfortunately, they are not.

Reply:
Ah, it's because this generic handling only kicks in for `list_of_objects` and `list_of_morphisms`. We could extend this to `IsObject` but this would break categories which actually have julia objects as object/morphism data (e.g. NemoLinearAlgebraForCAP).